sw/host/opentitanlib/src/test_utils/lc_transition.rs

```diff
@@ -90,31 +90,27 @@ pub fn trigger_lc_transition(
     if use_external_clk {
         jtag.write_lc_ctrl_reg(
             &LcCtrlReg::TransitionCtrl,
-            LcCtrlTransitionCtrlBit::union([LcCtrlTransitionCtrlBit::ExtClockEn]),
+            LcCtrlTransitionCtrl::EXT_CLOCK_EN.bits(),
         )?;
     } else {
         jtag.write_lc_ctrl_reg(&LcCtrlReg::TransitionCtrl, 0)?;
     }
 
     // Initiate LC transition and poll status register until transition is completed.
-    jtag.write_lc_ctrl_reg(
-        &LcCtrlReg::TransitionCmd,
-        LcCtrlTransitionCmdBit::union([LcCtrlTransitionCmdBit::Start]),
-    )?;
+    jtag.write_lc_ctrl_reg(&LcCtrlReg::TransitionCmd, LcCtrlTransitionCmd::START.bits())?;
     let one_millis = Duration::from_millis(1);
     loop {
         let status = jtag.read_lc_ctrl_reg(&LcCtrlReg::Status)?;
-        if (status & LcCtrlStatusBit::union([LcCtrlStatusBit::TransitionSuccessful])) != 0 {
+        let status =
+            LcCtrlStatus::from_bits(status).ok_or(LcTransitionError::InvalidState(status))?;
+
+        if status.contains(LcCtrlStatus::TRANSITION_SUCCESSFUL) {
             break;
         }
-        if (status
-            & !LcCtrlStatusBit::union([
-                LcCtrlStatusBit::Initialized,
-                LcCtrlStatusBit::Ready,
-                LcCtrlStatusBit::TransitionSuccessful,
-            ]))
-            != 0
-        {
+
+        if !status.contains(
+            LcCtrlStatus::INITIALIZED | LcCtrlStatus::READY | LcCtrlStatus::TRANSITION_SUCCESSFUL,
```

Comment:
I think this is a behavior change. The code used to return an error when STATUS contained any flags other than INITIALIZED, READY, or TRANSITION_SUCCESSFUL. Now it returns an error if status is missing one of those flags.

To preserve the original behavior, this should be something like this:

```rust
let expected_status = LcCtrlStatus::INITIALIZED | LcCtrlStatus::READY | LcCtrlStatus::TRANSITION_SUCCESSFUL;
if status != expected_status {
    return Err(...);
}
```

Reply:
Thanks, as above I've switched back to `!=` for these and will add a separate `ERRORS` check in the future if we need it.